tests/e2e/gauntlet_deployer.go

```diff
@@ -0,0 +1,370 @@
+	reportName := "ocr_deploy"
+	UpdateReportName(reportName, gd.Cli)
+	_, err = gd.Cli.ExecCommandWithRetries([]string{
+		"ocr2:deploy",
+		gd.Cli.Flag("rdd", rddPath),
+		gd.Cli.Flag("version", gd.Version),
+		tmpId,
+	}, []string{
+		TERRA_COMMAND_ERROR,
+	}, RETRY_COUNT)
+	Expect(err).ShouldNot(HaveOccurred(), "Failed to deploy ocr2")
+	ocrReport, err := LoadReportJson(reportName + ".json")
+	Expect(err).ShouldNot(HaveOccurred())
+	ocr := GetTxAddressFromReport(ocrReport)
+
+	// add the new contract to the rdd
+	ocrRddContract.Contracts[ocr] = ocrRddContract.Contracts[tmpId]
+	err = WriteRdd(ocrRddContract, rddPath)
+	Expect(err).ShouldNot(HaveOccurred(), "Did not write the rdd json correctly")
+	return ocr, rddPath
+}
+
+// SetBiling sets the billing info that exists in the rdd file for the ocr address you pass in
+func (gd *GauntletDeployer) SetBilling(ocr, rddPath string) {
+	UpdateReportName("set_billing", gd.Cli)
+	_, err := gd.Cli.ExecCommandWithRetries([]string{
+		"ocr2:set_billing",
+		gd.Cli.Flag("version", gd.Version),
+		gd.Cli.Flag("rdd", rddPath),
+		ocr,
+	}, []string{
+		TERRA_COMMAND_ERROR,
+	}, RETRY_COUNT)
+	Expect(err).ShouldNot(HaveOccurred(), "Failed to set billing")
+}
+
+// BeginProposal begins the proposal
+func (gd *GauntletDeployer) BeginProposal(ocr, rddPath string) string {
+	reportName := "begin_proposal"
+	UpdateReportName(reportName, gd.Cli)
+	_, err := gd.Cli.ExecCommandWithRetries([]string{
+		"ocr2:begin_proposal",
+		gd.Cli.Flag("version", gd.Version),
+		gd.Cli.Flag("rdd", rddPath),
+		ocr,
+	}, []string{
+		TERRA_COMMAND_ERROR,
+	}, RETRY_COUNT)
+	Expect(err).ShouldNot(HaveOccurred(), "Failed to begin proposal")
+	beginProposalReport, err := LoadReportJson(reportName + ".json")
+	Expect(err).ShouldNot(HaveOccurred())
+	return beginProposalReport["data"].(map[string]interface{})["proposalId"].(string)
+}
+
+// ProposeConfig proposes the config
+func (gd *GauntletDeployer) ProposeConfig(ocr, proposalId, rddPath string) {
+	reportName := "propose_config"
+	UpdateReportName(reportName, gd.Cli)
+	_, err := gd.Cli.ExecCommandWithRetries([]string{
+		"ocr2:propose_config",
+		gd.Cli.Flag("version", gd.Version),
+		gd.Cli.Flag("rdd", rddPath),
+		gd.Cli.Flag("f", "1"),
+		gd.Cli.Flag("proposalId", proposalId),
+		gd.OCR,
+	}, []string{
+		TERRA_COMMAND_ERROR,
+	}, RETRY_COUNT)
+	Expect(err).ShouldNot(HaveOccurred(), "Failed to propose config")
+}
+
+// ProposeOffchainConfig proposes the offchain config
+func (gd *GauntletDeployer) ProposeOffchainConfig(ocr, proposalId, rddPath string) {
+	reportName := "propose_offchain_config"
+	gd.Cli.NetworkConfig["SECRET"] = gd.Cli.NetworkConfig["MNEMONIC"]
+	UpdateReportName(reportName, gd.Cli)
+	_, err := gd.Cli.ExecCommandWithRetries([]string{
+		"ocr2:propose_offchain_config",
+		gd.Cli.Flag("version", gd.Version),
+		gd.Cli.Flag("rdd", rddPath),
+		gd.Cli.Flag("proposalId", proposalId),
+		gd.Cli.Flag("offchainConfigVersion", "2"),
```

Comment:
This is hardcoded on Gauntlet to be `2` always, so we can remove it

Reply:
fixed